_includes/how_to_apply.md

```diff
@@ -1,3 +1,21 @@
+## How To Apply
+
+{% if page.state == 'upcoming' %}
+{% include upcoming_statement.html %}
+{% else %}
+
+{% if page.state == 'closed' %}
+We're sorry, this job has closed.
+{% else %}
+Submit a complete online application prior to {{ page.closes | date: '%l:%M%P %Z on %A, %B %e, %Y' }}. Please fill out all applicable fields.
+
+<section class="usa-grid-full">
+  <a class="usa-button usa-button-secondary" href="{{ page.apply_url }}">Click here to apply</a>
+</section>
```

Comment:
I would suggest changing the button text to "Apply now" instead of using click here. 
I also think removing the `usa-button-secondary` class helps the button to stand out more on the page
<img width="593" alt="screen shot 2018-06-06 at 8 49 19 am" src="https://user-images.githubusercontent.com/1449852/41039072-5089e612-6966-11e8-9492-edb081a1f593.png">

Reply:
@rooneywp @JennMoran If you like those suggestions, happy to do them as a follow-up!